machine_learning_hep/computetrigger.py

```diff
@@ -27,9 +37,23 @@
 filedatamb = TFile.Open("/data/DerivedResults/D0kAnywithJets/vAN-20200304_ROOT6-1/pp_2018_data/376_20200304-2028/resultsMBvspt_ntrkl_trigger/masshisto.root")
```

Comment:
Ciao!
I didn't see this before, but we should not have paths hard-coded in the python code. The code in general should not depend on a certain state/configuration of a machine it's running on.

Reply:
Hi, this is nothing new, it was already in the code

Reply:
Ok, good to know. Because it will actually crash at some point in the future because this data is obsolete for D0 since last week. But then it's good we saw this :)

Reply:
Perfectly agree with that, i left the old values as the default of the main so as to keep track